backend/tracim_backend/lib/core/content.py

```diff
@@ -452,43 +460,28 @@ def create(
         content.workspace = workspace
         content.type = content_type.slug
         content.is_temporary = is_temporary
-        content.revision_type = ActionDescription.CREATION
-        content.content_namespace = content_namespace
-
         if template_id:
```

Comment:
Couldn't those lines be part of `copy_as_template`?

Reply:
probably. done.